battle/boss.py

```diff
@@ -702,26 +702,27 @@ def gui():
     ):
         logger.info("Gui2 - with extra Aeon overdrive")
         while memory.main.battle_active():
-            if screen.turn_seymour() and seymour_turn < 2:
-                battle.main.seymour_spell(target_face=False)
-                seymour_turn += 1
-            elif Yuna.is_turn() and seymour_turn >= 2:
-                logger.debug("Laser Time")
-                if memory.main.get_overdrive_battle(1) == 100:
-                    while not memory.main.other_battle_menu():
-                        xbox.tap_left()
-                    while not memory.main.interior_battle_menu():
-                        xbox.tap_b()
-                    while memory.main.interior_battle_menu():
-                        xbox.tap_b()
+            if memory.main.turn_ready():
+                if screen.turn_seymour() and seymour_turn < 2:
+                    battle.main.seymour_spell(target_face=False)
+                    seymour_turn += 1
+                elif Yuna.is_turn() and seymour_turn >= 2:
+                    logger.debug("Laser Time")
+                    if memory.main.get_overdrive_battle(1) == 100:
```

Comment:
A lot of the stuff in here can probably use the new player stuff. But we can sort that out later.

Reply:
oh haha this is all just tabbed right one level, I didn't even look at what it's doing. You are right though.

Reply:
Lol. That's fair. That also explains why this was left the old way and the new stuff added further down used the new way